this doesn't do anything except fallback to throw() on libc++ and fail
--- a/cupsfilters/pdftoraster.cxx
+++ b/cupsfilters/pdftoraster.cxx
@@ -2192,35 +2192,3 @@
   return (ret);
 #endif // HAVE_POPPLER
 }
-#ifdef HAVE_POPPLER
-
-// Replace memory allocation methods for memory check
-// For compatibility with g++ >= 4.7 compilers _GLIBCXX_THROW
-// should be used as a guard, otherwise use traditional definition
-#ifndef _GLIBCXX_THROW
-#define _GLIBCXX_THROW throw
-#endif
-
-void * operator new(size_t size) _GLIBCXX_THROW (std::bad_alloc)
-{
-  return malloc(size);
-}
-
-void operator delete(void *p) throw ()
-{
-  free(p);
-}
-
-void * operator new[](size_t size) _GLIBCXX_THROW (std::bad_alloc)
-{
-  return malloc(size);
-}
-
-#if 0
-void operator delete[](void *p) throw ()
-{
-  free(p);
-}
-#endif // 0
-
-#endif // HAVE_POPPLER
